packages/host/app/components/operator-mode/overlays.gts

```diff
@@ -292,5 +312,14 @@ export default class OperatorModeOverlays extends Component<Signature> {
     },
   );
 
-  // TODO: actions for 'preview' and 'more-actions' buttons
+  private loadRealmResource = task(async (card: CardDef) => {
+    let realmURL = await this.cardService.getRealmURL(card);
+    let resource = this.realmResources.get(realmURL.href);
+    if (!resource) {
+      resource = getRealm(this, { realmURL: () => realmURL });
```

Comment:
I'm trying to be efficient with the resources here since generally there only needs to be 1 realm resource per grid

Reply:
the lifetime of a resource is tied to the owner of a resource. it seems bad to share them unless you know they have the same owner (which is my case here)--otherwise the resource might go out of scope.